packages/trader/src/App/Components/Elements/ContractAudit/contract-details.jsx

```diff
@@ -57,49 +114,7 @@ const ContractDetails = ({ contract_end_time, contract_info, duration, duration_
                     value={localize('{{buy_value}} (Buy)', { buy_value: buy })}
                     value2={sell ? localize('{{sell_value}} (Sell)', { sell_value: sell }) : undefined}
                 />
-                {isMultiplierContract(contract_type) ? (
-                    <React.Fragment>
-                        <ContractAuditItem
-                            id='dt_commission_label'
-                            icon={<Icon icon='IcContractCommission' size={24} />}
-                            label={localize('Commission')}
-                            value={<Money amount={commission} currency={currency} show_currency />}
-                        />
-                        {!!cancellation_price && (
-                            <ContractAuditItem
-                                id='dt_cancellation_label'
-                                icon={<Icon icon='IcContractSafeguard' size={24} />}
-                                label={getLabel()}
-                                value={<Money amount={cancellation_price} currency={currency} />}
-                            />
-                        )}
-                    </React.Fragment>
-                ) : (
-                    <React.Fragment>
-                        <ContractAuditItem
-                            id='dt_duration_label'
-                            icon={<Icon icon='IcContractDuration' size={24} />}
-                            label={localize('Duration')}
-                            value={
-                                tick_count > 0
-                                    ? `${tick_count} ${tick_count < 2 ? localize('tick') : localize('ticks')}`
-                                    : `${duration} ${duration_unit}`
-                            }
-                        />
-                        <ContractAuditItem
-                            id='dt_bt_label'
-                            icon={
-                                isDigitType(contract_type) ? (
-                                    <Icon icon='IcContractTarget' size={24} />
-                                ) : (
-                                    <Icon icon='IcContractBarrier' size={24} />
-                                )
-                            }
-                            label={getBarrierLabel(contract_info)}
-                            value={getBarrierValue(contract_info) || ' - '}
-                        />
-                    </React.Fragment>
-                )}
+                {unic_card_info}
```

Comment:
```suggestion
                {unique_card_info}
```
@kate-deriv did you mean unique? )

Reply:
Yes, sorry about that

Reply:
no worries)